### PR TITLE
modify index.html and style.css files

### PR DESCRIPTION
**Pull request by GordonOtieno**
# modified index.html and style.css files.
added list attributes instead of divs
modified styles in style.css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,21 +11,31 @@
...
+          </li>
+          <li class="angelist-icon">
+            <a href="#"><img src="./icons/angel-list.png" alt="angelist"></a>
+          </li>
+          <li class="twiter-icon">
+            <a href="#"><img src="./icons/twitter.png" alt="twitter"></a>
+          </li>
+          <li class="medium-icon">
+            <a href="#"><img src="./icons/medium.png" alt="medium"></a>
+          </li>
```
Comment (ishimwezachee) [inline index.html:35]: - [ ]  You should use the same icons similar to the one from Figma design. 
Your Design 
image
Figma design 
image

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @GordonOtieno , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
LInters are passing successfully 
Project is running successfully 

 ## Required Changes ♻️

- [ ] Your readme documentation should have a description of action you've accomplished, and remember to add just one author with you active social medias 

image

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -4,6 +4,10 @@
 
 > Description the project.
 
+- This project is built with html and css.
+- Linters is properly set passing all the test.
+- Header is properly set with links
+- The design is for a mobile view.
```
Comment (gtekle) [inline README.md:10]: - [ ] The description section of the ReadMe should host the `description` of the project. That is, what the project is about briefly. Most of the ballpoints mentioned here should go to your `pull request summary` and the first ballpoint should go to the `Built With` section in the Readme.

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @GordonOtieno :wave: , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## I have seen that you have merged another pull request before it is approved. Please, make sure to not merge pull requests before they are approved by a code reviewer in upcoming projects!!
almost-there-01

 ## Highlights
:heavy_check_mark: Lingers Setup
:heavy_check_mark: Good use of Github flow
:heavy_check_mark: Nice UI

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly check the inline comments below.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @GordonOtieno , 
Good job so far!

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks fine:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 

 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**